site/data/sidebar.yml

```diff
@@ -92,6 +92,7 @@
 - title: Utilities
   pages:
     - title: API
+    - title: Background
     - title: Borders
     - title: Colors
```

Comment:
Shouldn't this be renamed as `Color` for consistency? Questionning only, no opinion.

Reply:
We actually seem to mix and match these... we have Interactions, shadows etc. We should probably streamline these at some point and add redirects.